Samples/DecoderSample/SensorDecoderModule.sln

```diff
@@ -1,25 +1,25 @@
 ﻿
 Microsoft Visual Studio Solution File, Format Version 12.00
-# Visual Studio 15
-VisualStudioVersion = 15.0.28010.2050
+# Visual Studio Version 16

```

Comment:
I think we don't need to change the solution version.

Reply:
Changed back.